### DISCUSSION TITLE
Support for associated types in trait

### DISCUSSION THREAD

**Discussion by marshtompsxd** [verus-lang/verus#379] [General]
Hi Verus developers, I am wondering whether there is any chance to support associated types in trait such as:
```
trait T {
    type A;
    type B;
}
```
Currently, Verus reports `error: The verifier does not yet support the following Rust feature: unsupported item`

**Comment by jaybosamiya** [comment]
In terms of prioritization, just adding a +1 here since support for associated types would help simplify the design of what I'm currently implementing.

**Reply by jonhnet** [reply]
Toss my interest on the pile -- Jay Lorch & I want it for the ironfleet port, too. We can work around the gap by making our <template<lists<really<long>>>>, though.

**Comment by utaal** [comment]
These are now supported.
